components/esp_websocket_client/esp_websocket_client.c

```diff
@@ -78,6 +81,14 @@ typedef struct {
     char                        *headers;
     int                         pingpong_timeout_sec;
     size_t                      ping_interval_sec;
+    const char                  *cert_pem;
```

Comment:
Rename it to remove the `_pem` part, since It's possible to have certificates in DER format.

Reply:
Yes, i will change it.